README.md

```diff
@@ -1,2 +1,103 @@
-# jest-preset
+# jest-config
+
+[![NPM version][npm-image]][npm-url] [![Downloads][downloads-image]][npm-url] [![Build Status][travis-image]][travis-url] [![Coverage Status][codecov-image]][codecov-url] [![Dependency status][david-dm-image]][david-dm-url] [![Dev Dependency status][david-dm-dev-image]][david-dm-dev-url]
+
+[npm-url]:https://npmjs.org/package/@moxy/jest-config
+[downloads-image]:https://img.shields.io/npm/dm/@moxy/jest-config.svg
+[npm-image]:https://img.shields.io/npm/v/@moxy/jest-config.svg
+[travis-url]:https://travis-ci.org/moxystudio/jest-config
+[travis-image]:http://img.shields.io/travis/moxystudio/jest-config/master.svg
+[codecov-url]:https://codecov.io/gh/moxystudio/jest-config
+[codecov-image]:https://img.shields.io/codecov/c/github/moxystudio/jest-config/master.svg
+[david-dm-url]:https://david-dm.org/moxystudio/jest-config
+[david-dm-image]:https://img.shields.io/david/moxystudio/jest-config.svg
+[david-dm-dev-url]:https://david-dm.org/moxystudio/jest-config?type=dev
+[david-dm-dev-image]:https://img.shields.io/david/dev/moxystudio/jest-config.svg
+
 MOXY's Jest configuration to be used across several JavaScript projects.
+
+
+## Installation
+
+```sh
+$ npm install @moxy/jest-config
+```
+
+
+## Usage
+
+Create `jest.config.js` at the root of your project:
+
+```js
+const { jestConfig } = require('@moxy/jest-config')
+
+module.exports = jestConfig();
+```
+
+## Addons
+
+This package comes with some preset addons that add further options to the Jest configuration to cover the needs of common situations. Here's a list of all addons we offer so far:
+
+| Addon | Description |
+|  ---   |     ---     |
+| [withWeb](addons/withWeb/withWeb.js)   | Adds setup and ignore patterns we use in [`next-with-moxy`](https://www.github.com/moxystudio/next-with-moxy) |
+
+### Instructions
+
+
+To use addons, use the `compose` function offered with this package. **Keep in mind**, the first item should always be the default configuration, `jestConfig`! Here's an example of using `compose`:
+
+```js
+const { compose, jestConfig, withWeb } = require('@moxy/jest-config');
+
+module.exports = compose([jestConfig, withWeb]);
+```
+
+If you want to make your own addon, you can! It just needs to have the following structure:
+
+```js
+const { compose, jestConfig } = require('@moxy/jest-config');
+
+// Receives previous configuration
+function myAddon(configuration) {
```

Comment:
Change to arrow function and keep the `withXXX` naming to be inline with the rest.